README.md

```diff
@@ -56,7 +56,8 @@ Using Gradle makes building WPILib very straightforward. It only has a few depen
     - If the WPILib installer was used, this toolchain is already installed
 - Raspberry Pi toolchain (optional)
     - Run `./gradlew installArm32Toolchain` after cloning this repository
-
+- SystemCore toolchain (required for SystemCore development)
+    - Run `./gradlew installSystemCoreToolchain` after cloning this repository
```

Comment:
This should have the same note as the roboRIO toolchain (if the 2027 wpilib installer is used this is already present)